src/main/java/com/google/devtools/build/lib/actions/SpawnResult.java

```diff
@@ -358,6 +363,15 @@ public String getDetailMessage(
       }
       return messagePrefix + " failed" + reason + explanation;
     }
+
+    @Nullable
+    @Override
+    public InputStream getInMemoryOutput(ActionInput output) {
```

Comment:
maybe `output` -> `outputFile` ?

Reply:
This just overrides the SpawnResult interface and there it's `output`. So I'll leave it at output.